generator/src/init.ts

```diff
@@ -94,6 +96,16 @@ export class Init {
                 const exists = await fs.pathExists(extensionPackagePath);
                 if (exists) {
                     const extensionPackage = await readPkg(extensionPackagePath);
+                    if (extensionPackage.prettier) {
```

Comment:
Actually, the prettier configuration seems to be found from che-theia/package.json: it seems the config resolution uses the resolved parent: there is no need to copy it to theia/package.json. Also: if there are conflicting values for the same key in various extensions packages, they would overwrite each other.

Reply:
1. well it won't when it's running from /projects/theia as with symlinks parent will be /projects/theia, not /projects/theia/che/che-theia  (sometimes, links are resolved sometimes no)

2. It may conflict but in reality there is only che-theia that is in fact cloned.